### ISSUE TITLE
Suggestion:  Earn an extra ball mechanism

### ISSUE THREAD

**Issue by cgku** [dozingcat/Vector-Pinball#135]
What a great pinball app.  Would love if there was a feature that would give let you earn another ball after X number of points earned; like seen on many traditional pinball machines.  Sometimes it would be nice to go more than three balls, but without having to resort to unlimited.

**Comment by dozingcat** [comment]
Good idea. There's support in the code for extra balls, but it's on a per-table basis and currently only one table has a way to get them. I'll keep it in mind for future tables.
